docs/topics/jvm/java-interop.md

```diff
@@ -43,9 +44,88 @@ fun calendarDemo() {
 }
 ```
 
+`calendar.firstDayOfWeek` above is a synthetic property.
+
 Note that, if the Java class only has a setter, it isn't visible as a property in Kotlin because Kotlin doesn't support
 set-only properties.
 
+## Java synthetic property references
+
+> This feature is [Experimental](components-stability.md#stability-levels-explained). It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We'd appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
```

Comment:
```suggestion
> We recommend that you use it only for evaluation purposes.
```
Danil asked me to not request feedback via YouTrack on experimental features in Docs. Only in what's new. So I guess the same applies here :)